tests/test-verify-pool.yml

```diff
@@ -15,6 +15,20 @@
     #     compression
     #     deduplication
 
+- name: Get VG shared value status
+  command: sudo vgs --noheadings --binary -o shared {{ storage_test_pool.name }}
```

Comment:
you shouldn't need to use `sudo` here - if you need to escalate privileges  for a task, use `become: true` https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_privilege_escalation.html#privilege-escalation-must-be-general

Reply:
Fixed.